src/async_fuse/memfs/s3_metadata.rs

```diff
@@ -504,7 +440,6 @@ impl<S: S3BackEnd + Sync + Send + 'static> MetaData for S3MetaData<S> {
                 );
             } else {
                 if let SFlag::S_IFREG = child_node.get_type() {
-                    self.local_mtime_and_size.remove(&child_ino);
```

Comment:
We maintain `open_count` in `open_files` now, so the `deferred_deletion` should refer to `open_files` but not `S3Node::open_count`, right?

Reply:
It just refer to `lookup_cnt` only.